src/LocalPath.hpp

```diff
@@ -17,40 +21,34 @@ namespace File {
  *
  * Throws on error.
  */
-void
-InitialiseDataPath();
+void InitialiseDataPath();
```

Comment:
Why you modify? There is no effect, just cosmetic.

Reply:
And its wrong code style in top of it.